### PR TITLE
Revise Installation>Requriements page

### LINKED ISSUE #314: Revise Installation>Requirements page
[Revise Harvester Overview page](https://docs.harvesterhci.io/v1.1/install/requirements)
- [ ] Revise page locally (https://github.com/LucasSaintarbor/harvester-docs)
  - [ ] Content adheres to [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice)
  - [ ] Headers are accurate
  - [ ] Lists are formatted properly (ordered vs unordered) 
  - [ ] Images are clear and sized appropriately
  - [ ] Validate links work and are accurate
- [ ] Open PR
- [ ] Ask for review from docs team member
- [ ] Close PR

#### Notes

### PR DESCRIPTION
**Pull request by LucasSaintarbor**
Resolves #314  

- Revised page for style/grammar based on [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice)
- Updated page and section titles 
- Updated formatting

### CODE REVIEW

File: docs/install/requirements.md [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
 sidebar_position: 1
-sidebar_label: Requirements
-title: "Requirements"
+sidebar_label: Hardware and Networking Requirements
+title: "Hardware and Networking Requirements"
 keywords:
 - Installation Requirements
 Description: Outline the Harvester installation requirements
 ---
+As an HCI solution on bare metal servers, there are minimum host hardware requirements and network requirements to install and run Harvester.
```
Comment (vickyhella) [inline docs/install/requirements.md]:
```suggestion
As an HCI solution on bare metal servers, there are minimum hardware and network requirements to install and run Harvester. 
```

Comment (vickyhella) [reply docs/install/requirements.md]: `servers` and `host` sound a little bit confusing

File: docs/install/requirements.md [hunk 2]
```diff
@@ -1,33 +1,41 @@
...
-:::info
-  We recommend server-class hardware for best results. Laptops and nested virtualization are not officially supported.
-:::
 
-## Networking
+## Networking Requirements
+
+Harvester nodes have the following network requirements for installation.
+
+### Inbound Rules for Harvester Hosts
```
Comment (vickyhella) [inline docs/install/requirements.md:35]:
```suggestion
### Inbound Rules for Harvester Nodes
```

Comment (vickyhella) [reply docs/install/requirements.md:35]: According to the context, I guess it should be `Nodes`?

File: docs/install/requirements.md [hunk 3]
```diff
@@ -61,18 +69,15 @@ To get the Harvester server up and running the following minimum hardware is req
...
 
-- K3s: [https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking)
-- RKE: [https://rancher.com/docs/rke/latest/en/os/#ports](https://rancher.com/docs/rke/latest/en/os/#ports)
-- RKE2: [https://docs.rke2.io/install/requirements#networking](https://docs.rke2.io/install/requirements#networking)
+### Inbound Rules for K3s or RKE/RKE2 clusters
 
+For the port requirements for guest clusters deployed inside Harvester VMs, refer to the following links.
+- [K3s](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking)
+- [RKE](https://rancher.com/docs/rke/latest/en/os/#ports)
+- [RKE2](https://docs.rke2.io/install/requirements#networking)
```
Comment (vickyhella) [inline docs/install/requirements.md:83]:
```suggestion
- [K3s Networking](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking)
- [RKE Ports](https://rancher.com/docs/rke/latest/en/os/#ports)
- [RKE2 Networking](https://docs.rke2.io/install/requirements#networking)
```

File: docs/install/requirements.md [hunk 4]
```diff
@@ -61,18 +69,15 @@ To get the Harvester server up and running the following minimum hardware is req
...
-#### Guest clusters
-As for the port requirements for the guest clusters deployed inside Harvester virtual machines, refer to the following links.
+The VMs of Kubernetes clusters that are provisioned from Rancher into Harvester also need to be able to connect to TCP port **443** of the Rancher load balancer. Otherwise, the cluster won't be manageable by Rancher. For more information, refer to [Rancher Architecture](https://rancher.com/docs/rancher/v2.6/en/overview/architecture/).
 
-- K3s: [https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking)
-- RKE: [https://rancher.com/docs/rke/latest/en/os/#ports](https://rancher.com/docs/rke/latest/en/os/#ports)
-- RKE2: [https://docs.rke2.io/install/requirements#networking](https://docs.rke2.io/install/requirements#networking)
+### Inbound Rules for K3s or RKE/RKE2 clusters
 
+For the port requirements for guest clusters deployed inside Harvester VMs, refer to the following links.
```
Comment (vickyhella) [inline docs/install/requirements.md]:
```suggestion
For the port requirements for guest clusters deployed inside Harvester VMs, refer to the following links:
```

File: docs/install/requirements.md [hunk 5]
```diff
@@ -61,18 +69,15 @@ To get the Harvester server up and running the following minimum hardware is req
...
-#### Guest clusters
-As for the port requirements for the guest clusters deployed inside Harvester virtual machines, refer to the following links.
+When provisioning VMs with Kubernetes clusters from Rancher into Harvester, you need to be able to connect to TCP port **443** of the Rancher load balancer. Otherwise, the cluster won't be manageable by Rancher. For more information, refer to [Rancher Architecture](https://rancher.com/docs/rancher/v2.6/en/overview/architecture/).
 
-- K3s: [https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking)
-- RKE: [https://rancher.com/docs/rke/latest/en/os/#ports](https://rancher.com/docs/rke/latest/en/os/#ports)
-- RKE2: [https://docs.rke2.io/install/requirements#networking](https://docs.rke2.io/install/requirements#networking)
+### Inbound Rules for K3s or RKE/RKE2 clusters
 
+Refer to the following links for the port requirements for guest clusters deployed inside Harvester VMs.
```
Comment (vickyhella) [inline docs/install/requirements.md:80]:
```suggestion
For the port requirements for guest clusters deployed inside Harvester VMs, refer to the following links:
```

**Review by vickyhella — COMMENTED** [review]
Only some nits ^^
And I notice this doc still uses title-style capitalization for Headings, please decide if we need to use sentence-style cap instead according to the Microsoft style guide

File: docs/install/requirements.md [hunk 6]
```diff
@@ -61,18 +69,15 @@ To get the Harvester server up and running the following minimum hardware is req
...
+If you want to [integrate Harvester with Rancher](../rancher/rancher-integration.md), you need to make sure that all Harvester nodes can connect to TCP port **443** of the Rancher load balancer.
 
-#### Guest clusters
-As for the port requirements for the guest clusters deployed inside Harvester virtual machines, refer to the following links.
+When provisioning VMs with Kubernetes clusters from Rancher into Harvester, you need to be able to connect to TCP port **443** of the Rancher load balancer. Otherwise, the cluster won't be manageable by Rancher. For more information, refer to [Rancher Architecture](https://rancher.com/docs/rancher/v2.6/en/overview/architecture/).
 
-- K3s: [https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking](https://rancher.com/docs/k3s/latest/en/installation/installation-requirements/#networking)
-- RKE: [https://rancher.com/docs/rke/latest/en/os/#ports](https://rancher.com/docs/rke/latest/en/os/#ports)
-- RKE2: [https://docs.rke2.io/install/requirements#networking](https://docs.rke2.io/install/requirements#networking)
+### Inbound Rules for K3s or RKE/RKE2 clusters
```
Comment (vickyhella) [inline docs/install/requirements.md:78]:
```suggestion
### Inbound Rules for K3s or RKE/RKE2 Clusters
```

**Comment by LucasSaintarbor** [comment]
> Only some nits ^^ And I notice this doc still uses title-style capitalization for Headings, please decide if we need to use sentence-style cap instead according to the Microsoft style guide

Thank you for reviewing @vickyhella! Added your suggestions - updated the header with sentence-style cap and applied the changes to v1.1.

**Review by vickyhella — APPROVED** [review]

**Pull request merged**